docker/Dockerfile.deps

```diff
@@ -119,5 +119,14 @@ RUN FFMPEG_VERSION=4.2.1 && \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" && make install && \
     cd /tmp && rm -rf ffmpeg-$FFMPEG_VERSION
 
+# libsnd
+RUN LIBSND_VERSION=1.0.28 && cd /tmp                                                                           && \
```

Comment:
Please add libsnd to the acknowledgments.  And update `docs/compilation.rst` with the similar note to FFmpeg one.

Reply:
Done